code-nautilus.py

```diff
@@ -9,15 +9,35 @@
 from subprocess import call
 import os
 
+
 # path to vscode
-VSCODE = 'code'
+FLATPAK_VSCODE_PATH = "/var/lib/flatpak/exports/bin/com.visualstudio.code"
+VSCODE_PATH = "/usr/bin/code"
+
+# path to vscodium
+FLATPAK_VSCODIUM_PATH = "/var/lib/flatpak/exports/bin/com.vscodium.codium"
+VSCODIUM_PATH = "/usr/bin/codium"
+
+# VSCODENAME: what name do you want to see in the context menu? 
 
-# what name do you want to see in the context menu?
-VSCODENAME = 'Code'
+if os.path.exists(FLATPAK_VSCODE_PATH):
+    VSCODE = "flatpak run com.visualstudio.code"
+    VSCODENAME = 'Code'
+elif os.path.exists(FLATPAK_VSCODIUM_PATH):
+    VSCODE = "flatpak run com.vscodium.codium"
+    VSCODENAME = 'VSCodium'
+elif os.path.exists(VSCODE_PATH):
+    VSCODE = 'code'
+    VSCODENAME = 'Code'
+elif os.path.exists(VSCODIUM_PATH):
+    VSCODE = 'codium'
+    VSCODENAME = 'VSCodium'
```

Comment:
If someone has VSCodium installed and VSCode, chances are that one prefers codium?
At least that is the case for me :)